packages/react-app/src/views/BuilderProfileView.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect, useState } from "react";
+import { Typography } from "antd";
+import { Link, useParams } from "react-router-dom";
+import axios from "axios";
+import ChallengeList from "../components/ChallengeList";
+import Address from "../components/Address";
+
+const { Title } = Typography;
+
+export default function BuilderProfileView({ serverUrl, mainnetProvider }) {
+  const { builderAddress } = useParams();
+
+  const [builder, setBuilder] = useState();
+
+  useEffect(() => {
+    async function fetchBuilder() {
+      const fetchedBuilder = await axios.get(serverUrl + `builders/${builderAddress}`);
+      setBuilder(fetchedBuilder.data);
+      console.log(fetchedBuilder.data);
+    }
+    fetchBuilder();
+  }, [builderAddress]);
+
+  return (
+    <div>
+      <Link to="/builders">{"<"}Back to list of builders</Link>
+      <Title>Builder Progress</Title>
+      {builder ? (
+        <>
+          <Address address={builder.id} ensProvider={mainnetProvider} />
+          <div style={{ textAlign: "start" }}>
+            <ChallengeList userChallenges={builder.challenges} />
+          </div>
+        </>
+      ) : null}
```

Comment:
Let's use a spinner instead of `null`. Not sure if we already have a custom scaffold spinner, but we can use [the antd default](https://ant.design/components/spin/) if we haven't. With a quick search in the repo I haven't found a custom one.